src/locales/pt.json

```diff
@@ -0,0 +1,217 @@
+{
+  "Blockchain": {
+    "Mnemonic": "Mnemônico",
+    "Address": "Endereço",
+    "Wallet": "Carteira",
+    "Wallets": "Carteiras",
+    "Coins": "Moedas",
+    "Tokens": "Tokens",
+    "Balance": "Saldo",
+    "Tx hash": "Hash Tx",
+    "Gas": "Gas",
```

Comment:
Is it `Gas` or `Gás`?

Reply:
I avoided to translate this as it is a word used in the context of crypto transactions.

Reply:
We can check here in this blog post how this word usually does not get translated to portuguese
https://academy.bit2me.com/pt/o-que-%C3%A9-g%C3%A1s-em-ethereum/